azure_li_services/network.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2017 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of azure-li-services.
+#
+# azure-li-services is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# azure-li-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with azure-li-services.  If not, see <http://www.gnu.org/licenses/>
+#
+import ipaddress
+from textwrap import dedent
+
+# project
+from azure_li_services.exceptions import AzureLiNetworkConfigDataException
+
+
+class AzureLiNetworkSetup(object):
```

Comment:
Using Li in names that covers implementation for Li and VLi may lead to confusion at some point going forward when we need methods that are specific to one or the other type. It may be advantageous to use a generic name for such cases, maybe "AzureHosted", "AzureSAPEnv", "AzureMetal", "AzurePhys'

Reply:
yes agreed, I like AzureHosted, will change it